src/components/BottomSheet.tsx

```diff
@@ -16,6 +16,7 @@ interface Props {
   buttonLabel: string
   buttonOnPress: () => void
   buttonType?: BtnTypes
+  buttonLoading?: boolean
```

Comment:
Side note, I think I missed the PR for this `BottomSheet` but slightly surprised it always has a button.
Seems this could have been grouped into an optional `button` element passed to it which allows for greater customization.

Reply:
yes good point...i abstracted this a bit hastily. i want to leave it like this for this PR and i'll discuss with Reed how we want to use the sheet going forward, and update the component. 

perhaps we should have a BottomSheet that just takes children, and also have an ActionBottomSheet that contains the title / description / button. or just have a BottomSheet that takes children....idk. but preferably not a BottomSheet that takes optional title / description / button / children, that feels like the first 3 props are useless...